portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,91 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#pf-image {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
+
+h1, h2 {
+  display: flex;
+  justify-content: center;
+  text-transform: uppercase;
+}
+
+.icon-row {
+  overflow: auto;
+  text-align: center;
+  width: 100%;
+}
+
+.image-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.single-container, .image-container img {
+  max-height: 200px;
+  max-width: 200px;
+}
+
+.single-container {
+  margin: 10px;
+  position: relative;
+}
+
+.slide-container:hover .img-block, .single-container:hover .img-block {
+  opacity: .70;
 }
+
+.img-block {
+  background-color: blue;
+  bottom: 0;
+  height: 25%;
+  left: 0;
+  opacity: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  transition: .5s ease;
+  width: 100%;
+}
+
+.img-caption {
+  -webkit-transform: translate(-50%, -50%);
```

Comment:
Would it be equivalent to set `top` and `left` to `0` and then remove the transform? CSS positioning frightens and confuses me, so it seems very well possible that it would not be equivalent, but I'm wondering.

Reply:
just tried doing this -- replacing -webkit-transform with top = 0 and left = 0 seems to move the img-caption element out of bounds, to the top left corner.